boards/nxp/imx95_evk/CMakeLists.txt

```diff
@@ -11,3 +11,11 @@ if (CONFIG_SOF AND CONFIG_BOARD_IMX95_EVK_MIMX9596_M7_DDR)
     DEPENDS ${CMAKE_BINARY_DIR}/zephyr/${KERNEL_ELF_NAME}
   )
 endif()
+
+
+if (CONFIG_BOARD_IMX95_EVK_MIMX9596_M7 AND CONFIG_BOARD_NXP_SPSDK_IMAGE)
+  set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+    COMMAND cp -r ${ZEPHYR_HAL_NXP_MODULE_DIR}/zephyr/blobs/mimx9596/imx95-19x19-lpddr5-evk ${CMAKE_BINARY_DIR}/zephyr/
+    COMMAND nxpimage ahab export -c ${PROJECT_SOURCE_DIR}/boards/nxp/imx95_evk/imx95_evk_mimx9596_m7_ahab.yaml
```

Comment:
this is not going to work on windows since cp is a *nix command

Reply:
Will point to source path without copying.